ldscript.txt

```diff
@@ -161,7 +161,7 @@ SECTIONS
         build/sa2/src/game/interactables_2/notification_ring_bonus.o(.text);
         build/sa2/src/game/interactables_2/sky_canyon_init.o(.text);
         build/sa2/src/game/interactables_2/105.o(.text);
-        build/sa2/src/game/interactables_2/note_particle.o(.text);
```

Comment:
For now I am following the linker for where things should be placed. Based on that this isn't in the music plant folder (as it's not grouped with the other music plant stuff)

Reply:
I've undone this so I can merge for my changes